frontend/src/types.ts

```diff
@@ -635,6 +635,27 @@ export const ProgressionStepTitles = {
   [ProgressionStep.SERVER_STARTED]: 'Server started',
 };
 
+export const AssociatedSteps = {
+  [ProgressionStep.NOTEBOOK_CONTAINER_STARTED]: [
+    ProgressionStep.SERVER_REQUESTED,
+    ProgressionStep.POD_CREATED,
+    ProgressionStep.POD_ASSIGNED,
+    ProgressionStep.PVC_ATTACHED,
+    ProgressionStep.INTERFACE_ADDED,
+    ProgressionStep.PULLING_NOTEBOOK_IMAGE,
+    ProgressionStep.NOTEBOOK_IMAGE_PULLED,
+    ProgressionStep.NOTEBOOK_CONTAINER_CREATED,
+  ],
```

Comment:
```suggestion
  [ProgressionStep.NOTEBOOK_CONTAINER_STARTED]: [
    ProgressionStep.PVC_ATTACHED,
    ProgressionStep.INTERFACE_ADDED,
    ProgressionStep.PULLING_NOTEBOOK_IMAGE,
    ProgressionStep.NOTEBOOK_IMAGE_PULLED,
    ProgressionStep.NOTEBOOK_CONTAINER_CREATED,
  ],
```
This kind of separation is better because the "pod" and "server requested" are technically not tied to the Notebook Container (pod is the wrapper, and server requested is always available)